filebeat/prospector/config.go

```diff
@@ -13,38 +13,34 @@ import (
 
 var (
 	defaultConfig = prospectorConfig{
-		Enabled:        true,
-		DocumentType:   "log",
-		IgnoreOlder:    0,
-		ScanFrequency:  10 * time.Second,
-		InputType:      cfg.DefaultInputType,
-		CleanInactive:  0,
-		CleanRemoved:   true,
-		HarvesterLimit: 0,
-		Symlinks:       false,
-		TailFiles:      false,
+		Enabled:       true,
+		DocumentType:  "log",
+		ScanFrequency: 10 * time.Second,
+		InputType:     cfg.DefaultInputType,
+		CleanRemoved:  true,
```

Comment:
I strongly prefer to have all values in here, even if they are the golang defaults. This makes it very easy (and future proof) to lookup what our defaults are without having to think about what the default in Golang is.